rclc/include/rclc/executor.h

```diff
@@ -0,0 +1,387 @@
+// Copyright (c) 2019 - for information on the respective copyright owner
+// see the NOTICE file and/or the repository https://github.com/ros2/rclc.
+// Copyright 2014 Open Source Robotics Foundation, Inc.
```

Comment:
are you sure that this should be copyrighted by OSRF?

Reply:
Yes. OSRF started the development of rclc.